redis-utils/src/main/java/org/ballerinalang/redis/actions/string/Set.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http:www.apache.orglicensesLICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.redis.actions.string;
+
+import org.ballerinalang.jvm.values.HandleValue;
+import org.ballerinalang.redis.BallerinaRedisDbException;
+import org.ballerinalang.redis.RedisDataSource;
+import org.ballerinalang.redis.actions.AbstractRedisAction;
+
+import static org.ballerinalang.redis.Constants.REDIS_EXCEPTION_OCCURRED;
+
+/**
+ * {@code Set} Maps with "Set" operation of Redis.
+ *
+ * @since 0.5.0
+ */
+public class Set extends AbstractRedisAction {
```

Comment:
Instead of having separate classes for `get`, `set`, `append`, shall we have a single class with set of static methods. Group them into a single class on relevant basis,

Reply:
Updated in c0668f3